routes/teams.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable prefer-destructuring */
+/* eslint-disable consistent-return */
+/* eslint-disable no-console */
+const express = require('express');
+const bodyParser = require('body-parser');
+const mysql = require('mysql');
+
+const router = express.Router();
+
+router.use(bodyParser.urlencoded({ extended: true }));
+router.use(bodyParser.json());
+
+const connection = mysql.createConnection({
```

Comment:
You still need to use environment variables to mask the database information. Look at server.js for reference. .env file is in the slack

Reply:
I'll look into it

Reply:
Fixed this issue in latest commit.